src/azure-cli/requirements.py3.windows.txt

```diff
@@ -109,15 +109,15 @@ jmespath==0.9.5
 jsondiff==1.3.0
 knack==0.9.0
 MarkupSafe==1.1.1
-msal-extensions==0.3.0
+msal-extensions==0.3.1
 msal==1.16.0
 msrest==0.6.21
 msrestazure==0.6.3
 oauthlib==3.0.1
 packaging==21.3
 paramiko==2.6.0
 pbr==5.3.1
-portalocker==1.7.1
+portalocker==2.3.2
```

Comment:
Also bump `portalocker` as new version is now supported by `msal-extensions`: https://github.com/AzureAD/microsoft-authentication-extensions-for-python/pull/97